content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -436,192 +623,275 @@ Resources are deleted in two phases: 1) finalization, and 2) removal.
-  updated, not how up-to-date the object is when served.
-- **Continue Token, Exact:** Return data at the resource version of the initial paginated list
-  call. The returned Continue Tokens are responsible for keeping track of the initially provided
-  resource version for all paginated list calls after the initial paginated list call.
-
-For watch, the semantics of resource version are:
-
-**Watch:**
+
+Most recent
+: Return data at the most recent resource version. The returned data must be
+  consistent (in detail: served from etcd via a quorum read).
+
+Not older than
+: Return data at least as new as the provided `resourceVersion`. The newest
+  available data is preferred, but any data not older than the provided `resourceVersion` may be
+  served.  For **list** requests to servers that honor the `resourceVersionMatch` parameter, this
+  guarantees that the collection's `.metadata.resourceVersion` is not older than the requested
+  `resourceVersion`, but does not make any guarantee about the `.metadata.resourceVersion` of any
+  of the items in that collection.
+
+Exact
+: Return data at the exact resource version provided. If the provided `resourceVersion` is
+  unavailable, the server responds with HTTP 410 "Gone".  For **list** requests to servers that honor the
+  `resourceVersionMatch` parameter, this guarantees that the collection's `.metadata.resourceVersion`
+  is the same as the `resourceVersion` you requested in the query string. That guarantee does
+  not apply to the `.metadata.resourceVersion` of any items within that collection.
+
+Continue Token, Exact
+: Return data at the resource version of the initial paginated **list** call. The returned _continue
+  tokens_ are responsible for keeping track of the initially provided resource version for all paginated
+  **list** calls after the initial paginated **list**.
+
+{{< note >}}
+When you **list** resources and receive a collection response, the response includes the
+[metadata](/docs/reference/generated/kubernetes-api/v1.21/#listmeta-v1-meta) of the collection as
+well as [object metadata](/docs/reference/generated/kubernetes-api/v1.21/#listmeta-v1-meta)
+for each item in that collection. For individual objects found within a collection response,
+`.metadata.resourceVersion` tracks when that object was last updated, and not how up-to-date
+the object is when served.
+{{< /note >}}
+
+When using `resourceVersionMatch=NotOlderThan` and limit is set, clients must
+handle HTTP 410 "Gone" responses. For example, the client might retry with a
+newer `resourceVersion` or fall back to `resourceVersion=""`.
+
+When using `resourceVersionMatch=Exact` and `limit` is unset, clients must
+verify that the collection's `.metadata.resourceVersion` matches
+the requested `resourceVersion`, and handle the case where it does not. For
+example, the client might fall back to a request with `limit` set.
+
+### Semantics for **watch**
+
+For **watch**, the semantics of resource version are:
+
```

Comment:
Check lines 892 - 894, and then lines started at line 904, The meaning of those **watch** semantics are:
I think there may be some duplication,
https://deploy-preview-25299--kubernetes-io-main-staging.netlify.app/docs/reference/using-api/api-concepts/#semantics-for-watch

Reply:
It's not very clear; maybe a follow-up PR could try to make this easier to read?